Issue #2245 Infinite sheds beam fraction on ground zenith guardrail syntax bug (#2359)
pvlib/pvlib-python: docs/sphinx/source/whatsnew/v0.11.3.rst

Enhancements
~~~~~~~~~~~~

> Fix syntax for unshaded ground fraction (:issue:`2245`, :pull:`2359`)

Reviewer: ```suggestion
Fix bug in :py:func:`pvlib.bifacial.get_irradiance_poa` which may had yielded non-zero ground irradiance with sun below horizon (:issue:`2245`, :pull:`2359`)

```

Enhancements
~~~~~~~~~~~~

> Fix bug in :py:func:`pvlib.bifacial.get_irradiance_poa` which may had yielded non-zero ground
> irradiance with sun below horizon (:issue:`2245`, :pull:`2359`)

Reviewer 2: ```suggestion
* Fix a bug in :py:func:`pvlib.bifacial.get_irradiance_poa` which may have yielded non-zero
  ground irradiance when the sun was below the horizon. (:issue:`2245`, :pull:`2359`)
```
